### PR TITLE
feat: add verification stage for non-store channels.(OK-30064)

### PR DESCRIPTION
**Pull request by huhuanming**

### CODE REVIEW

**Comment by huhuanming** [comment]
@SocketSecurity ignore npm/protobufjs@6.11.4
@SocketSecurity ignore npm/web3@1.10.3
@SocketSecurity ignore npm/biginteger@1.0.3
@SocketSecurity ignore npm/electron@31.2.0

**Comment by huhuanming** [comment]
@SocketSecurity ignore npm/es5-ext@0.10.64

File: apps/desktop/src-electron/process/AutoUpdate.ts [hunk 1]
```diff
@@ -22,6 +28,59 @@ interface ILatestVersion {
   isManualCheck: boolean;
 }
 
+const signingKey = `-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (originalix) [inline apps/desktop/src-electron/process/AutoUpdate.ts]: 放一个新的文件里，并且这个文件跳过拼写检查呢

CleanShot 2024-07-16 at 17 13 34@2x

Comment (huhuanming) [reply apps/desktop/src-electron/process/AutoUpdate.ts]: okay

Comment (huhuanming) [reply apps/desktop/src-electron/process/AutoUpdate.ts]: 弄好啦

**Review by originalix — APPROVED** [review]

**Review by revan-zhang — APPROVED** [review]

**Pull request merged**